### PR TITLE
Feature/performance fixes

### PR DESCRIPTION
**Pull request by mosquito**
A little bit performance fixes

### CODE REVIEW

File: aiormq/channel.py [hunk 1]
```diff
@@ -560,6 +563,21 @@ async def basic_reject(
         if drain_future is not None:
             await drain_future
 
+    def _split_body(self, body: bytes) -> List[ContentBody]:
+        if not body:
+            return []
+
+        if len(body) < self.max_content_size:
+            return [ContentBody(body)]
```
Comment (dizballanze) [inline aiormq/channel.py:571]: I think this is unnecessary.

File: aiormq/channel.py [hunk 2]
```diff
@@ -560,6 +563,21 @@ async def basic_reject(
...
+
+        result = []
+        with io.BytesIO(body) as fp:
+            data = fp.read(self.max_content_size)
+            while data:
+                result.append(ContentBody(data))
+                data = fp.read(self.max_content_size)
```
Comment (dizballanze) [inline aiormq/channel.py:578]: It's better to remove duplication:
```python
while True:
    data = fp.read(self.max_content_size)
    if not data:
        break
    result.append(ContentBody(data))
```

File: aiormq/channel.py [hunk 3]
```diff
@@ -560,6 +563,21 @@ async def basic_reject(
         if drain_future is not None:
             await drain_future
 
+    def _split_body(self, body: bytes) -> List[ContentBody]:
+        if not body:
+            return []
```
Comment (dizballanze) [inline aiormq/channel.py:568]: This check is also redundant.

**Review by dizballanze — CHANGES_REQUESTED** [review]

File: aiormq/channel.py (hunk 1 above)
Comment (mosquito) [reply aiormq/channel.py:571]: Thats optimization

**Pull request merged**